app/controllers/api/base_controller/renderer.rb

```diff
@@ -205,7 +217,7 @@ def expand_virtual_attributes(json, type, resource)
       def fetch_direct_virtual_attribute(type, resource, attr)
         return unless attr_accessible?(resource, attr)
         virtattr_accessor = virtual_attribute_accessor(type, attr)
```

Comment:
Not sure how `virtattr_accessor ( send(virtattr_accessor, resource) ` is used. 
Should it be also in RBAC ? What are examples of using it ?